requirements/test.in

```diff
@@ -3,15 +3,15 @@
 blockbuster
 coverage
 freezegun
-isal
+isal; python_version < "3.14" # no wheel for 3.14
```

Comment:
This isn't worth doing. Basic dependencies (like multidict) will not have 3.14 compatible releases until atleast the release candidate is out. We don't really want a load of version checks that need to be removed again shortly.